File: Backend/AngularWiki/AngularWiki.API/Models/Entities/Database/Article.cs
using Microsoft.EntityFrameworkCore;
using Microsoft.EntityFrameworkCore.Metadata.Builders;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AngularWiki.API.Models.Entities.Database
{
    public class Article
    {
        public long? Id { get; set; }

        public string Title { get; set; }

        public string FirstParagraph { get; set; }

        public string Body { get; set; }

        public string ImageUrl { get; set; }

        public DateTime? CreatedOn { get; set; }

        public DateTime? ModifiedOn { get; set; }
    }

    public class ArticleDbConfig : IEntityTypeConfiguration<Article>
    {
        public void Configure(EntityTypeBuilder<Article> builder)
        {
            builder.ToTable("ArticleTable");

            builder.HasKey(x => x.Id);

            builder.Property(x => x.Title).HasColumnType("nvarchar(256)").IsRequired();
            builder.Property(x => x.FirstParagraph).HasColumnType("nvarchar(256)").IsRequired();
            builder.Property(x => x.Body).HasColumnType("nvarchar(max)");
            builder.Property(x => x.ImageUrl).HasColumnType("nvarchar(255)");

            builder.Property(x => x.CreatedOn).HasColumnType("datetime2");
            builder.Property(x => x.ModifiedOn).HasColumnType("datetime2");
        }
    }
}

File: ClientApp/src/app/models/entities/Article.ts
import { IArticle } from './IArticle';
import { ITag } from './ITag';

export class Article implements IArticle {
    id: number;    
    title: string;
    firstParagraph: string;
    body: string;
    imageUrl: string;
    tags: ITag[];

    constructor(article: IArticle) {
        this.id = article.id;
        this.title = article.title;
        this.firstParagraph = article.firstParagraph;
        this.body = article.body;
        this.imageUrl = article.imageUrl;
        this.tags = article.tags;
    }

    public getTagsString(): string {
        if (this.tags && this.tags.length > 0){
            return this.tags
                .map(t => t.title)
                .join(", ");
        }
        else {
            return "";
        }
    }
}